### DISCUSSION TITLE
Supported Minecraft server cores

### DISCUSSION THREAD

**Discussion by GamerVII-NET** [Gml-Launcher/Gml.Backend#80] [Announcements]
## Welcome to our community! :wave:

Our project is currently in the active development and testing phase. At this moment, testing was conducted on a limited number of cores and the information about bugs or limitations might be incomplete or inaccurate.

We greatly appreciate any help you can offer. :heart: If you have experience in setting up or successfully running a game server with our project, we'd love to hear from you. Please share your observations and suggestions in this discussion thread.

Your contribution will help us improve our project and make it more reliable and efficient. :+1:

**Comment by Arsenii1109** [comment]
Омагад поко

**Comment by SH1SHOCHKA** [comment]
![2024-07-11_20-56-37.jpg](https://github.com/user-attachments/assets/053f1d8f-363b-40b8-bfd4-c4b2879a135b)


**Whoever moves first is gay**

**Reply by multivarka1** [reply]
![2D-among-us-Senko-(Sewayaki-Kitsune-no-Senko-san)-Sewayaki-Kitsune-no-Senko-san-Anime-5750739](https://github.com/user-attachments/assets/09a05a07-150e-4e31-8739-99b30dfe88ac)
